.github/workflows/main.yml

```diff
@@ -15,17 +15,17 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Build Docker Container
-        run: docker compose -f docker-compose.yml build >/dev/null
+        run: docker compose -f docker-compose.yml build --build-arg PHP_VERSION=${{matrix.php}} >/dev/null
         env:
           PHP_VERSION: ${{matrix.php}}
 
       - name: Composer Install
-        run: docker compose -f docker-compose.yml run -e PHP_VERSION=${{matrix.php}} test_runner composer install --no-progress
```

Comment:
The php version variable doesn't relevant at this point - we're never installing composer packages for old php versions.